src/main/java/org/apache/commons/io/output/FileWriterWithEncoding.java

```diff
@@ -59,7 +57,7 @@
      * @return the initialized writer
      * @throws IOException if an error occurs
      */
-    private static Writer initWriter(final File file, final Object encoding, final boolean append) throws IOException {
+    private static final Writer initWriter(final File file, final Object encoding, final boolean append) throws IOException {
```

Comment:
This one is not really necessary @wodencafe ? `initWriter` is already private, so making it `final` won't change much for internal/external devs?

Reply:
I usually do this out of habit, but after doing some research it seems that there is no functional difference for adding the `final` modifier to a private static nested class. It has been removed.